### PR TITLE
Update xunit version on dotnet/arcade release branch

### PR DESCRIPTION
**Pull request by maryamariyan**
Cherry pick of https://github.com/dotnet/arcade/pull/7911

Resolved conflicts:
	eng/Versions.props

### CODE REVIEW

**Review by Anipik — APPROVED** [review]

File: src/Microsoft.DotNet.Helix/Sdk/tools/xunit-runner/XUnitRunner.targets [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
   <ItemGroup Condition="'@(XUnitProject)' != ''">
     <HelixCorrelationPayload Include="xunit-runner">
-      <Uri>https://api.nuget.org/v3-flatcontainer/xunit.runner.console/$(XUnitRunnerVersion)/xunit.runner.console.$(XUnitRunnerVersion).nupkg</Uri>
+      <Uri>https://api.nuget.org/v3-flatcontainer/xunit.runner.console/2.4.1/xunit.runner.console.$(XUnitRunnerVersion).nupkg</Uri>
```
Comment (maryamariyan) [inline src/Microsoft.DotNet.Helix/Sdk/tools/xunit-runner/XUnitRunner.targets:15]:
```suggestion
      <Uri>https://dev.azure.com/dnceng/public/_packaging?_a=package&feed=dotnet-eng&package=xunit.runner.console&protocolType=NuGet&version=$(XUnitRunnerVersion)</Uri>
```

File: src/Microsoft.DotNet.Helix/Sdk/tools/xunit-runner/XUnitRunner.targets [hunk 2]
```diff
@@ -12,7 +12,7 @@
 
   <ItemGroup Condition="'@(XUnitProject)' != ''">
     <HelixCorrelationPayload Include="xunit-runner">
-      <Uri>https://api.nuget.org/v3-flatcontainer/xunit.runner.console/$(XUnitRunnerVersion)/xunit.runner.console.$(XUnitRunnerVersion).nupkg</Uri>
+      <Uri>https://dev.azure.com/dnceng/public/_packaging?_a=package&feed=dotnet-eng&package=xunit.runner.console&protocolType=NuGet&version=$(XUnitRunnerVersion)</Uri>
```
Comment (maryamariyan) [inline src/Microsoft.DotNet.Helix/Sdk/tools/xunit-runner/XUnitRunner.targets:15]:
```suggestion
      <Uri>https://pkgs.dev.azure.com/dnceng/9ee6d478-d288-47f7-aacc-f6e6d082ae6d/_packaging/1a5f89f6-d8da-4080-b15f-242650c914a8/nuget/v3/flat2/xunit/$(XUnitRunnerVersion)/xunit.$(XUnitRunnerVersion).nupkg</Uri>
```

File: src/Microsoft.DotNet.Helix/Sdk/tools/xunit-runner/XUnitRunner.targets [hunk 3]
```diff
@@ -12,7 +12,7 @@
 
   <ItemGroup Condition="'@(XUnitProject)' != ''">
     <HelixCorrelationPayload Include="xunit-runner">
-      <Uri>https://api.nuget.org/v3-flatcontainer/xunit.runner.console/$(XUnitRunnerVersion)/xunit.runner.console.$(XUnitRunnerVersion).nupkg</Uri>
+      <Uri>https://pkgs.dev.azure.com/dnceng/9ee6d478-d288-47f7-aacc-f6e6d082ae6d/_packaging/1a5f89f6-d8da-4080-b15f-242650c914a8/nuget/v3/flat2/xunit/$(XUnitRunnerVersion)/xunit.$(XUnitRunnerVersion).nupkg</Uri>
```
Comment (ericstj) [inline src/Microsoft.DotNet.Helix/Sdk/tools/xunit-runner/XUnitRunner.targets:15]:
```suggestion
      <Uri>https://pkgs.dev.azure.com/dnceng/9ee6d478-d288-47f7-aacc-f6e6d082ae6d/_packaging/1a5f89f6-d8da-4080-b15f-242650c914a8/nuget/v3/flat2/xunit.runner.console/$(XUnitRunnerVersion)/xunit.runner.console.$(XUnitRunnerVersion).nupkg</Uri>
```

**Review by Anipik — APPROVED** [review]

**Comment by maryamariyan** [comment]
@ericstj I don't have merge privileges here.

**Comment by ViktorHofer** [comment]
Me neither. ping @markwilkie

**Comment by markwilkie** [comment]
done.

**Pull request merged**